src/Notifications.jsx

```diff
@@ -95,22 +94,17 @@ export class Notifications extends PureComponent {
     }
 
     componentWillReceiveProps({ isShowing, isVisible, wpcom }) {
-        initAPI(wpcom);
-
-        if (this.props.isShowing && !isShowing) {
-            store.dispatch(actions.ui.closePanel());
-            // unselect the note so keyhandlers don't steal keystrokes
-            reset();
```

Comment:
With our current implementation we still need this since we will steal keystrokes if a note is left open and we close the panel. See: https://github.com/Automattic/notifications-panel/pull/106

Basically open a post/comment note, close the panel. We won't be able to type `l` or any of the other bound keys in the Calypso editor/text fields.